.gitignore

```diff
@@ -103,3 +103,7 @@ ENV/
 _site/
```

Comment:
This should be the last set, as they are specific to this repository.

.idea and other common problems should appear earlier.

Reply:
Done. 😄